src/components/common/Tab/TabItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { cls } from '@/utils'
+
+interface TabItemProps {
+  active?: boolean
+  text?: string
+  onClick: (_e?: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const TabItem = ({ active, text, onClick }: TabItemProps) => {
+  return (
+    <div
+      onClick={() => {
+        onClick()
+      }}
+      className={cls(
+        active ? 'border-emerald-500' : 'border-slate-300',
```

Comment:
```suggestion
        active ? 'border-emerald5' : 'border-slate3,
```
border에 다크 모드 적용이 필요합니다! 🌙